src/simulate/config/props.ini

```diff
@@ -0,0 +1,3 @@
+[oauth]
```

Comment:
Although `logging.config.fileConfig()` requires a `.ini` file, that format doesn't need to be used for all of this app's config files.  Consider using YAML for this file instead.

Reply:
Yes, I realized that yaml is better and changed it in the next PR #14